tests/data/strings.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0" ?>
+<!-- Test translation file.
+   - Any copyright is dedicated to the Public Domain.
+   - http://creativecommons.org/publicdomain/zero/1.0/ -->
+<!DOCTYPE resources [ <!ENTITY foo "Foo"> <!ENTITY bar "Bar &foo;"> ]>
+<resources>
+  <string name="one"></string>
+  <string name="two"/>
+  <!-- bar -->
+  <string name="three">value</string>
+  <!-- bar -->
+  <!-- foo -->
+  <string name="four">multi-line comment</string>
+  <string name="five" translatable="false">@string/three</string>
+
+  <!-- standalone -->
+
+  <string name="welcome">Welcome to <b>&foo;</b>!</string>
+  <string name="placeholders">Hello, %1$s! You have %2$d new messages.</string>
+  <string name="escape_html">Hello, %1$s! You have &lt;b>%2$d new messages&lt;/b>.</string>
+
+  <string name="ws_trimmed"> &#32; &#8200; &#8195;</string>
+  <string name="ws_quoted">" &#32; &#8200; &#8195;&quot;</string>
+  <string name="ws_escaped"> \u0032 \u8200 \u8195</string>
+
+  <string name="control_chars">\u0000 \u0001</string>
+
+  <string name="foo"><![CDATA[Foo Bar <a href=\"foo?id=%s\">baz</a> is cool]]></string>
+  <string name="busy">Sorry, &foo; is &lt;i>not available&lt;/i> just now.</string>
+
+  <string-array name="planets_array">
+    <item>Mercury</item>
+    <item>Venus</item>
+    <item>Earth</item>
+    <item>Mars</item>
+  </string-array>
+
+  <plurals name="numberOfSongsAvailable">
+    <!--
+         As a developer, you should always supply "one" and "other"
+         strings. Your translators will know which strings are actually
+         needed for their language. Always include %d in "one" because
+         translators will need to use %d for languages where "one"
+         doesn't mean 1 (as explained above).
```

Comment:
Where "above"?

Reply:
Ah, a remnant from the [Android docs](https://developer.android.com/guide/topics/resources/string-resource.html#Plurals) from where I lifted this.